Bugsnag/Assets/Bugsnag/Editor/SymbolUpload/BugsnagAndroidSymbolUploader.cs

```diff
@@ -0,0 +1,246 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Runtime.InteropServices;
+using UnityEditor;
+using UnityEditor.Build;
+using UnityEditor.Build.Reporting;
+using UnityEngine;
+
+namespace BugsnagUnity.Editor
+{
+
+    public class BugsnagAndroidSymbolUploader : IPostprocessBuildWithReport
+    {
+        public int callbackOrder => 1;
+        private const string CLI_DOWNLOAD_VERSION = "2.6.2";
+        private string _cliDownloadPath = Path.Combine(Application.dataPath, $"../bugsnag/bin/v{CLI_DOWNLOAD_VERSION}/bugsnag_cli");
```

Comment:
It looks like they'll get a build-up of CLI downloads over time if we encode the version in the path?

Should we get more sophisticated here? Calling `bugsnag_cli --version` first or storing the version in the text file?

Reply:
I could just remove any directories that are named after different versions than the one expected?